finetuner/client/client.py

```diff
@@ -168,7 +169,13 @@ def get_run_status(self, experiment_name: str, run_name: str) -> dict:
             run_name,
             STATUS,
         )
-        return self._handle_request(url=url, method=GET)
+
+        try:
+            response = self._handle_request(url=url, method=GET, timeout=10)
+        # In the event that the request times out, attempt the request one more time.
+        except ReadTimeout:
+            response = self._handle_request(url=url, method=GET, timeout=10)
+        return response
```

Comment:
So basically, we don't really know why it is failing but trying to get the status again if it is not working?

Reply:
Yes :/ . I reran the ci a couple of times until I got a read timeout and trying again like this succeeded. Its not ideal but at least this makes the ci more reliable